docs/glossary.md

```diff
@@ -384,6 +384,9 @@ HAProxy
 nginx
     [nginx](https://docs.nginx.com/nginx/) (pronounced "engine x") is an HTTP and reverse proxy server, a mail proxy server, and a generic TCP/UDP proxy server, originally written by Igor Sysoev.
 
+traefik
```

Comment:
This is a proper name and there are many products under the Traefik brand name. We should be explicit to disambiguate it from all their other products.

`Traefik` should also be added to https://github.com/plone/documentation/blob/6.0/styles/Vocab/Plone/accept.txt

```suggestion
Traefik Proxy
```